src/config.rs

```diff
@@ -388,6 +388,8 @@ impl Config {
             // GOG Galaxy:
             (format!("{}/GOG Galaxy/Games", pf32), Store::GogGalaxy),
             (format!("{}/GOG Galaxy/Games", pf64), Store::GogGalaxy),
+            // GOG Linux:
+            ("~/GOG Games".to_string(), Store::GogLinux),
```

Comment:
Nice :+1: Always good to have more of these known locations.

Reply:
Dropped since already included in Store::Gog